core/src/main/java/com/exadel/core/services/impl/ShopContentServiceImpl.java

```diff
@@ -18,10 +20,14 @@
 public class ShopContentServiceImpl implements ShopContentService {
 
     private static final String PAGE_PATH = "/content/exadel/us/en/products";
+    private static final String TAG_PATH = "/content/cq:tags/exadel/brands/";
 
     @Reference
     private ResourceResolverFactory resolverFactory;
 
+    /**
+     * Creates new product page with product component and corresponding attributes
+     */
```

Comment:
Please create separate methods for each object creation. E.g. createPage, createRootNode, etc.